src/Migration/Sources/Appwrite.php

```diff
@@ -460,6 +463,7 @@ protected function exportGroupDatabases(int $batchSize, array $resources)
             $this->addError(
                 new Exception(
                     Resource::TYPE_DATABASE,
+                    Transfer::GROUP_DATABASES,
                     $e->getMessage()
                 )
```

Comment:
We should pass `$e` as `previous` everywhere we can, that way the stack trace will include the trace from the original exception too

Reply:
Better to keep them separate; we might not want to defer every message/code to `previous`